MapboxCoreNavigation/Constants.swift

```diff
@@ -152,3 +152,6 @@ public var RouteControllerMinimumNumberLocationUpdatesBackwards = 3
 public var RouteControllerNumberOfSecondsForRerouteFeedback: TimeInterval = 10
 
 let FasterRouteFoundEvent = "navigation.fasterRoute"
+
+let MGLErrorDomain = "MGLErrorDomain"
```

Comment:
This constant needs to be public so developers can conditionalize logic in exception handlers based on it. It also needs a little documentation.